packages/queued-request-controller/README.md

```diff
@@ -0,0 +1,15 @@
+# `@metamask/queued-request-controller`
+
+Includes a controller and middleware that implements a request queue
```

Comment:
Nit: Add a period at the end?

``` suggestion
Includes a controller and middleware that implements a request queue.
```

Would also be nice to explain what a request queue does, or why it's necessary at some point.

Reply:
Ive added a bit more to the readme, still is pretty barebones but its a start!